src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java

```diff
@@ -0,0 +1,414 @@
+            FakeModularUIContainerClipboard fakeModularUIContainer = new FakeModularUIContainerClipboard(ui, this);
+            this.guiContainerCache = fakeModularUIContainer;
+            this.guiCache = new FakeModularGui(ui, fakeModularUIContainer);
+            this.writeCustomData(1, buffer -> { });
+        } catch (Exception e) {
+            GTLog.logger.error(e);
+        }
+    }
+
+
+    @Override
+    protected void initializeInventory() {
+        super.initializeInventory();
+        this.itemInventory = new InaccessibleItemStackHandler();
+    }
+
+    public ItemStack getClipboard() {
+        if (this.itemInventory.getStackInSlot(0) == ItemStack.EMPTY) {
+            ((InaccessibleItemStackHandler) this.itemInventory).setStackInSlot(0, CLIPBOARD.getStackForm());
+        }
+        return this.itemInventory.getStackInSlot(0);
+    }
+
+    public void setClipboard(ItemStack stack) {
+        ((InaccessibleItemStackHandler) this.itemInventory).setStackInSlot(0, stack.copy());
+    }
+
+    @Override
+    public void getDrops(NonNullList<ItemStack> dropsList, @Nullable EntityPlayer harvester) {
+        dropsList.clear();
+        dropsList.add(this.getClipboard());
+    }
+
+    @Override
+    public float getBlockHardness() {
+        return 100;
+    }
+
+    @Override
+    public int getHarvestLevel() {
+        return 4;
+    }
+
+    @Override
+    public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing facing, CuboidRayTraceResult hitResult) {
+        if (!playerIn.isSneaking()) {
+            if (getWorld() != null && !getWorld().isRemote) {
+                MetaTileEntityUIFactory.INSTANCE.openUI(getHolder(), (EntityPlayerMP) playerIn);
+            }
+        } else {
+            BlockPos pos = this.getPos(); // Saving this for later so it doesn't get mangled
+            World world = this.getWorld(); // Same here
+            NonNullList<ItemStack> drops = NonNullList.create();
+            getDrops(drops, playerIn);
+
+            world.playEvent(2001, pos, Block.getStateId(world.getBlockState(pos)));
+            Block.spawnAsEntity(playerIn.world, this.getPos(), drops.get(0));
+            this.dropAllCovers();
+            this.onRemoval();
+
+            world.removeTileEntity(pos);
+            world.setBlockState(pos, Blocks.AIR.getDefaultState(), 3);
+        }
+        return true;
+    }
+
+
+    @Override
+    public String getHarvestTool() {
+        return "axe";
+    }
+
+    @Override
+    public void addCollisionBoundingBox(List<IndexedCuboid6> collisionList) {
+        collisionList.add(new IndexedCuboid6(null, GTUtility.rotateAroundYAxis(CLIPBOARD_AABB, EnumFacing.NORTH, this.getFrontFacing())));
+    }
+
+    public IndexedCuboid6 getPageCuboid() {
+        return new IndexedCuboid6(null, GTUtility.rotateAroundYAxis(pageBox.aabb(), EnumFacing.NORTH, this.getFrontFacing()));
+    }
+
+    @Override
+    public Pair<TextureAtlasSprite, Integer> getParticleTexture() {
```

Comment:
This seems to be failing when you shift click to remove it, creating "no texture" particles. I cannot replicate it when doing things that normally cause particles to appear, such as running and jumping